server/src/test/java/io/crate/protocols/postgres/types/PGArrayTest.java

```diff
@@ -133,6 +133,28 @@ public void test_can_decode_unquoted_strings_which_do_not_contain_numbers() thro
         ));
     }
 
+    @Test
+    public void test_can_decode_unquoted_words_with_multiple_spaces() throws Exception {
+        String s = "{ a b c , d e f }";
```

Comment:
If you think it's not too much maybe something like this:
```
String s = "{ \ta   b c  , \t d e   f  \t }";
```

```
assertThat(values, contains(
    "a   b c",
    "d e   f"
));
```

Reply:
Good point about tabs - I forgot that we can use them in the beginning of the unquoted tokens due to WS skip rule. Thanks!

Reply:
thx!